src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -256,6 +256,16 @@ def _validate_vm_create_nics(namespace):
 
 def _validate_vm_create_auth(namespace):
 
+    if len(namespace.admin_username) < 6 or namespace.admin_username.lower() == 'root':
```

Comment:
Are we sure this is what CRP does? / Is this the same validation the portal does?